### DISCUSSION TITLE
[预览]v1.12.11

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2007] [分享 / Share]
- 修复`瓜子换硬币`失效. (#1988)
- 修复`专栏图片导出`的命名出现`-1`. (#1973)
- 修复直播页面中`快速收起评论区`等动态功能失效. (#1900)
- 进一步完善专栏的夜间模式. (#1991)
- 修复`收起未关注人消息`打开时, `自定义顶栏`的消息数目仍然包含未关注人. (#1985)
- 分区数据更新, 影响包括`自定义顶栏`和`简化首页`. (#1511, #1918)
- `自定义顶栏`增加`番剧`入口. (#1938)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.11-preview'>[预览]v1.12.11</a>.</em>

**Comment by RiSeikiqwq** [comment]
正常用户能看的番剧不能下载了

**Reply by RingingVolcano** [reply]
是B站灰度的新版播放器造成的 之前已经有好几个issue反馈了
